src/stories/index.js

```diff
@@ -0,0 +1,24 @@
+/* global document */
+// better to load this only if it's needed
+import 'document-register-element'; // ES2015
+// load this for browsers which support customElements without builtin (webkit)
+import '@ungap/custom-elements-builtin';
+import { storiesOf } from '@storybook/html';
+
+import globalStyles from '../index.scss';
```

Comment:
We could either use webpack for this or [`preview-head.html` ](https://storybook.js.org/configurations/add-custom-head-tags/)(though only css not scss)